content/fair/interoperable/index.md

```diff
@@ -0,0 +1,141 @@
+
+Base URLs (home pages): https://usegalaxy.org, https://usegalaxy.eu, https://usegalaxy.org.au
+
+Introductory page URL: https://galaxyproject.org
+
+### Does Galaxy have a community **recognition**? What about inter-organisational collaborations, geographical diversity in the source of the submissions, international diversity of delivery partners and/or funders?
+
+Recognition of Galaxy is broad and observed across multiple communities. Galaxy as a platform originates from the US (usegalaxy.org) and has been developed by a worldwide community since its initial release in 2005. [The Galaxy community](https://galaxyproject.org/community/) is traditionally open, welcoming, diverse and highly active. With Galaxy Europe and Galaxy Australia another two major instances have been founded, complemented with multiple national ones, several of them located in Europe.
+
+#### Galaxy Main
+
+#### Galaxy Europe
+For example on usegalaxy.eu, more than 83,000 users are registered, every month another 1000 new users join. Here, >1 million computational jobs are launched per month, as well as ~180,000 workflows runs, and >100,000,000 datasets are aggregated to date. Galaxy Europe reaches users from 100+ countries and is used by companies, universities, publicly funded independent research institutions and citizen science projects. 
+
+The European Galaxy server is one of the largest ELIXIR services and part of several past and running implementation Studies (IS). For multiple EOSC projects, usegalaxy.eu is the preferred gateway for scientific computing, making it an ideal resource for researchers from a broad range of scientific fields. As a European resource, we address users from all over Europe, but also attract researchers from a global scope. Moreover, in the EOSC project ‘[EuroScienceGateway](https://galaxyproject.org/projects/esg/)’, started in September 2022, a central mission is to link usegalaxy.eu with 12+ national endpoints of the [Pulsar compute network](https://pulsar-network.readthedocs.io/en/latest/) for improved job re-distribution across Europe.
+Analogously, the team behind usegalaxy.eu builds from a number of groups and individuals, distributed all over Europe and of broad international backgrounds. 
+
+#### Galaxy Australia
+
```

Comment:
[Galaxy Australia](https://usegalaxy.org.au/) is a fully subsidised national service, delivered collaboratively through national and partner organisation funding. The service is hosted by Australian Academic and Research Network (AARNet) with data [Pulsar](https://pulsar-network.readthedocs.io/en/latest/) nodes at the two national Supercomputing Centers; NCI and Pawsey, as well as the University of Melbourne, the Queensland Cyber Infrastructure Foundation (QCIF) and on Azure. This combination of Pulsars allows users to run jobs requiring high-memory instances, GGPUs alongside all other tool resource requirements.
Galaxy Australia hosts two Galaxy Labs; Genomes and Proteomics, tailored as a focal point for best-practice analytics in genome assembly / annotation and proteomics, respectively.